packages/localizations/src/en-US.ts

```diff
@@ -2,6 +2,26 @@ import type { LocalizationResource } from '@clerk/types';
 
 export const enUS: LocalizationResource = {
   locale: 'en-US',
+  taskSelectOrganization: {
+    signOut: {
+      actionLink: 'Sign out',
+      actionText: 'Signed in as {{identifier}}',
+    },
+    createOrganization: {
+      title: 'Setup your account',
+      subtitle: 'Tell us a bit about your organization',
+      formButtonSubmit: 'Create organization',
+      formButtonReset: 'Cancel',
+    },
+    selectOrganization: {
+      title: 'Choose an account',
+      subtitle: 'Select the account with which you wish to continue',
+      suggestionsAcceptedLabel: 'Pending approval',
+      action__createOrganization: 'Create organization',
```

Comment:
```suggestion
      action__createOrganization: 'Create new organization',
```

As per Colin feedback.